rest/model/data/meta.go

```diff
@@ -16,6 +16,13 @@ type FeedPtr struct {
 	FeedID string `json:"feed,omitempty"`
 }
 
+// PulsarMeta is currently only used for validation
+type PulsarMeta struct {
+	JobID     string  `json:"job_id,omitempty"`
+	Bias      string  `json:"bias,omitempty"`
+	A5MCutoff float64 `json:"a5m_cutoff,omitempty"`
+}
```

Comment:
there's a "granularities" arg documented in API docs that send me down a bit of a rabbit hole before I figured out it's been disabled intentionally (PLS-61)